### ISSUE TITLE
New CRAN release?

### ISSUE THREAD

**Issue by etiennebacher** [extendr/rextendr#362]
There are several nice features in the devel version, is there a planned date for the next CRAN release?

**Comment by CGMossa** [comment]
I _think_ we can first hope for a response next week on this. But you're absolutely right! I also think we need to stabilise the extendr release, and hopefully, this will happen soon. Here, I mean that we need to publish patch-release to 0.7.0, because we found a few things to bugfix.

**Comment by Ilia-Kosenkov** [comment]
Yep, we will make a release. I will clean things up since I just returned from a vacation.

**Comment by JosiahParry** [comment]
Okay folks! I think we need a new CRAN release with helpers for addressing CRAN support of extendr packages as well. 

There are a few things that we need to accomplish with this. 

- [ ] set minimum version of R to R >= 4.1 https://github.com/extendr/rextendr/issues/375
- [ ] Introduce MSRV checks to configure 
    - see related work in https://github.com/albersonmiranda/fio/pull/85 and https://github.com/R-ArcGIS/arcpbf/pull/5/files#diff-dfe1640e6faf2820d725a02843f204ea6684ee30966a1fab5b72a177957e494c
- [ ] add function to support Fedora GHA checks which is the most common source of failing CRAN checks in my experience 
    - see related https://github.com/R-ArcGIS/arcpbf/pull/5/files#diff-cf424ae0341f100cdd07d3750b291b6deedb7c48b2d755881063883673fdd8d5

**Comment by Ilia-Kosenkov** [comment]
@JosiahParry , check also the milestone I defined some time ago. https://github.com/extendr/rextendr/milestone/4
Should we tackle those before proceeding or should we just postpone/discard?

**Comment by JosiahParry** [comment]
I think we should do those too. I think we also need to set the minimum version of R in the DESCRIPTION file to 4.1 via `usethis::use_package("R", "Depends", "4.1.")`

**Comment by kbvernon** [comment]
I can help implement `use_msrv()` if no one else is working on it yet.

**Comment by schochastics** [comment]
Hope this fits here:
My package [timeless](https://cran.r-project.org/web/packages/timeless/index.html) was archived which I can only assume comes from this warning:
```
Version: 0.2.4
Check: Rust compilation
Result: WARN
    Downloads Rust crates
    No rustc version reported prior to compilation
Flavors: [r-devel-linux-x86_64-fedora-clang](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-clang/timeless-00check.html), [r-devel-linux-x86_64-fedora-gcc](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-fedora-gcc/timeless-00check.html)
```

I got an email about this prior. I had submitted an updated version which specified the required rust version in the `SystemRequirements` field like `use_msrv()` does, but I guess it was not enough? Is there another place where the rust version needs to go? If so, I guess this should be reflected in `use_msrv()`

**Comment by Ilia-Kosenkov** [comment]
Hey @schochastics , we have a Discord where CRAN is discussed among other things. One of the team members forwarded your comment there, feel free to join if you'd like. Check [extendr README](https://github.com/extendr/extendr?tab=readme-ov-file#contributing) for instructions.
